docs/reference/docs/index_.asciidoc

```diff
@@ -86,7 +86,9 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=refresh]
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=routing]
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
+`timeout`::
+Period to wait for a response. If no response is received before the timeout
```

Comment:
This is really the same timeout as under `bulk`, I suggest to copy the same text here.